pom.xml

```diff
@@ -652,4 +652,4 @@
             </plugin>
         </plugins>
     </reporting>
-</project>
\ No newline at end of file
+</project>
```

Comment:
Don't touch this line. Minimize necessary changes.

Reply:
It is caused by the editor I used.
I will make sure that it won't happen again.